src/main/resources/blockly/js/mcreator_blocks.js

```diff
@@ -507,6 +507,12 @@ Blockly.Extensions.register('potion_list_provider',
             arrayToBlocklyDropDownArray(javabridge.getListOf("potion"))), 'potion');
     });
 
+Blockly.Extensions.register('potion_type_list_provider',
+    function () {
+        this.appendDummyInput().appendField(new Blockly.FieldDropdown(
+            arrayToBlocklyDropDownArray(javabridge.getListOf("potiontype"))), 'potiontype');

```

Comment:
Since `potion` is still being used by the effect list provider, should this remain `potiontype`?

Reply:
If there is no chance that both providers would be used in a same procedure block, than it can be potion. Otherwise, maybe call it potionitem?